docs/concepts/tools/kubectl/object-management-using-commands.md

```diff
@@ -0,0 +1,155 @@
+For example, the Service object has several subtypes including ClusterIP,
+LoadBalancer, and NodePort. Here's an example that creates a Service with
+subtype NodePort:
+
+```shell
+kubectl create service nodeport <myservicename>
+```
+
+In the preceding example, the `create service nodeport` command is called
+a subcommand of the `create service` command.
+
+You can use the `-h` flag to find the arguments and flags supported by
+a subcommand:
+
+```shell
+kubectl create service nodeport -h
+```
+
+## How to update objects
+
+The `kubectl` command supports verb-driven commands for some common update operations.
+These commands are named to enable users unfamiliar with Kubernetes
+objects to perform updates without knowing the specific fields
+that must be set:
+
+- `scale`: Horizontally scale a controller to add or remove Pods by updating the replica count of the controller.
+- `annotate`: Add or remove an annotation from an object.
+- `label`: Add or remove a label from an object.
+
+The `kubectl` command also supports update commands driven by field name.
+These commands require the user to know the names of the fields
+they want to update:
+
+- `set` <field>: Set the value of a specific field of an object.
+
+**Note**: In Kubernetes version 1.5, not every verb-driven command has an
+associated field-driven command.
+
+The `kubectl` tool supports these additional ways to update a live object directly,
+however they require a better understanding of the Kubernetes object schema.
+
+- `edit`: Directly edit the raw configuration of a live object by opening its configuration in an editor.
+- `patch`: Directly modify specific fields of a live object by using a patch string. 
+For more details on patch strings, see the patch section in
+[API Conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#patch-operations).
+
+## How to delete objects
+
+You can use the `delete` command to delete an object from a cluster:
+
+- `delete <type>/<name>`
+
+**Note**: You can use `kubectl delete` for both imperative commands and imperative object
+configuration. The difference is in the arguments passed to the command. To use
+`kubectl delete` as an imperative command, pass the object to be deleted as
+an argument. Here's an example that passes a Deployment object named nginx:
+
+```shell
+kubectl delete deployment/nginx
+```
+
+## How to view an object
+
+You can use `kubectl view` to print specific fields of an object.
+
+- `view`: Prints the value of a specific field of an object.
+
+There are several more commands for printing additional information about
+an object:
+
+- `get`: Prints basic information about matching objects.  Use `get -h` to see a list of options.
+- `describe`: Prints aggregated detailed information about matching objects.
+- `logs`: Prints the stdout and stderr for a container running in a Pod.
+- `top`: Print machine resource usage (cpu, memory) for a Kubernetes object.
```

Comment:
not sure this belongs in a general user guide... normal users likely won't have access to the metrics endpoints `top` assumes